docs/notebooks/u_mass_tutorial.ipynb

```diff
@@ -0,0 +1,173 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Demonstration for the `u_mass` topic coherence using topic coherence pipeline"
```

Comment:
@tmylk I've added an example notebook for `u_mass` topic coherence. Is this alright?